src/main/scala/com/monsanto/arch/cloudformation/model/simple/Builders.scala

```diff
@@ -244,6 +181,18 @@ trait Subnet extends AvailabilityZone with Outputs {
 
     f(sub) ++ sub.andOutput(name, name)
   }
+
+  def nat(routeTables: Seq[`AWS::EC2::RouteTable`])(implicit s: `AWS::EC2::Subnet`) = {
```

Comment:
The Builder for EC2 currently takes an implicit subnet and we are routinely using implicit VPCs. I'm fairly neutral on which way we go though.